tests/test_manifests.py

```diff
@@ -22,53 +28,35 @@
 
 def test_product_manifest_properties():
     """Test that all models inheriting from ProductModel have a .manifest property
-    And that it generates valid JSON. TODO: Use a library to generate + validate the SPDX data"""
-    product = ProductFactory()
-    version = ProductVersionFactory()
-    stream = ProductStreamFactory()
-    variant = ProductVariantFactory(name="1")
-    pnode = ProductNode.objects.create(parent=None, obj=product, object_id=product.pk)
-    pvnode = ProductNode.objects.create(parent=pnode, obj=version, object_id=version.pk)
-    psnode = ProductNode.objects.create(parent=pvnode, obj=stream, object_id=stream.pk)
-    _ = ProductNode.objects.create(parent=psnode, obj=variant, object_id=variant.pk)
-    # This generates and saves the product_variants property of stream
-    stream.save_product_taxonomy()
-    assert variant.name in stream.product_variants
-
-    build = SoftwareBuildFactory(
-        build_id=1,
-        completion_time=datetime.strptime("2017-03-29 12:13:29 GMT+0000", "%Y-%m-%d %H:%M:%S %Z%z"),
-    )
-    component = ComponentFactory(
-        software_build=build,
-        type="SRPM",
-        product_variants=[variant.ofuri],
-        product_streams=[stream.ofuri],
-    )
-    _, _ = component.cnodes.get_or_create(
-        type=ComponentNode.ComponentNodeType.SOURCE, parent=None, purl=component.purl
-    )
-    # The product_ref here is a variant name but below we use it's parent stream
-    # to generate the manifest
-    ProductComponentRelationFactory(
-        build_id="1", product_ref="1", type=ProductComponentRelation.Type.ERRATA
-    )
-
-    build.save_component_taxonomy()
-    build.save_product_taxonomy()
+    And that it generates valid JSON."""
+    component, stream, provided, dev_provided = setup_products_and_components()
 
     manifest = json.loads(stream.manifest)
 
+    # From https://raw.githubusercontent.com/spdx/spdx-spec/development/
+    # v2.2.2/schemas/spdx-schema.json
+    with open("tests/data/spdx-22-spec.json", "r") as spec_file:
+        schema = json.load(spec_file)
+    try:
+        jsonschema.validate(manifest, schema)
+    except jsonschema.exceptions.ValidationError as e:
+        print(e)
+        assert False
```

Comment:
```suggestion
    jsonschema.validate(manifest, schema)
```

Reply:
Ok, I'll change this to just throw the error instead of an assertion/failure.